README.md

```diff
@@ -74,7 +79,8 @@ These can be run by:
 
 ## [^](#code-written-by-john-baxter)How to use
 If you want to try out this game app, the following steps are required.
-(Instructions are specific to using a MacOS, other interfaces may require different/analogous steps.)
+(Instructions are specific to using a MacOS, other interfaces may require different/analogous steps.)\
+You will need to have Python 3 installed.
```

Comment:
Be more specific with the Python version. Just major version, not minor. I.e. python 3.8